pkgs/by-name/si/sile/package.nix

```diff
@@ -0,0 +1,181 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
```

Comment:
Almost for any package these come first.

Reply:
I'm a bit puzzled what the issue with ZST is here and why my version was tripping up. It seems to work fine [in the Libertinus font package](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/libertinus/default.nix).

Reply:
> I'm a bit puzzled what the issue with ZST is here and why my version was tripping up. It seems to work fine [in the Libertinus font package](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/libertinus/default.nix?rgh-link-date=2024-11-04T11%3A17%3A44Z).

I know, unpacking zst tarballs is pretty much supported in Nixpkgs. The problem I experienced was with the `fetchCargoTarball` function that did not know how to handle those zst tarballs, giving out unclear errors..

Reply:
I'm glad I'm not the only one confused by that error! I wonder how hard that would be to fix (I'm sue the error handling is way above my pay grade, that's hard stuff, I'm just thinking of knowing how to uncompress a ztd file).